public_html/wp-content/mu-plugins/blocks/assets/src/sponsors/sponsor-select.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * External dependencies.
+ */
+import { includes } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+const { Component }       = wp.element;
+const { __ } = wp.i18n;
+
+
+import ItemSelect from '../shared/item-select';
+
+function SponsorOption( option ) {
+	let sponsorOption;
+
+	if ( 'post' === option.type ) {
+		sponsorOption = SponsorPostOption( option );
+	} else {
+		sponsorOption = SponsorLevelOption( option );
+	}
```

Comment:
`MIGHT` - This could be simplified to a 1-line ternary.

Reply:
This has been eliminated in favor of the abstracted `Option` element that is part of the shared `ItemSelect` component. 3ac487368582f47135bed827bbb4b9f8ffa2b04d